records_mover/url/base.py

```diff
@@ -176,9 +176,9 @@ def open(self, mode: str = "rb") -> IO[Any]:
         "exist in the directory."
         raise NotImplementedError()
 
-    def wait_to_exist(self):
-        "Returns true after the file exists--useful for eventually consistent stores (e.g., S3)"
-        return True
+    def wait_to_exist(self) -> None:
+        "Returns after the file exists--useful for eventually consistent stores (e.g., S3)"
+        return
```

Comment:
Whoops, I didn't have a consistent output type here.  Nothing uses the output, thankfully, so easy to make consistent!